src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -3092,7 +3092,7 @@ public static void DCS_MyPersonSurrogate()
     [Fact]
     public static void DCS_FileStreamSurrogate()
     {
-        const string TestFileName = "Test.txt";
+        string TestFileName = Path.GetTempFileName();
```

Comment:
This is going to leave behind the temp file after the test completes. We should either a) add a try/finally with the finally deleting TestFileName, or b) use a base class like https://github.com/dotnet/corefx/blob/master/src/Common/tests/System/IO/FileCleanupTestBase.cs and its GetTestFilePath() helper in order to have it take care of the deletion automatically.